user-registration-app/src/app/views/live-workshop/live-workshop.component.html

```diff
@@ -14,30 +14,15 @@
       <a href="{{ download }}" style="color: #e5704c">Download {{ i + 1 }}</a>
     </p>
   </div>
-  <div class="inline">
-    <div
-      class="card-image tooltipped"
+  <div>
+    <b>Presenters:</b>
+  </div>
+  <div *ngFor="let presenterName of (presenters || [])" class="inline">
+    <p>{{presenterName}}</p>
+    <div class="card-image tooltipped" data-position="bottom"
       [ngStyle]="{ background: 'url(' + presenter1img + ')' }"
```

Comment:
Should presenter1img be changed since it's removed in the corresponding ts file?